doc/installationguide.md

```diff
@@ -254,6 +254,9 @@ overrides.
 | IOTA_MONGO_PORT           | `mongodb.port`                  |
 | IOTA_MONGO_DB             | `mongodb.db`                    |
 | IOTA_MONGO_REPLICASET     | `mongodb.replicaSet`            |
+| IOTA_MONGO_USER 	    | `mongodb.user`		      |
+| IOTA_MONGO_PASSWORD	    | `mongodb.password`	      |
+| IOTA_MONGO_AUTH_DB 	    | `mongodb.authDB`		      |
```

Comment:
```suggestion
| IOTA_MONGO_AUTH_DB 	    | `mongodb.authDb`		      |
```

Not fully sure... we use camelCase as convention, but in the case of acronyms (and DB is an acronym here) I'm not sure how to apply it. But looking other cases (e.g. `clientId`) I think is better this way :)

Reply:
I renamed authDB to authSource. So it is directly clear what is set in the connection string by the MongoDB.